build/de.cau.cs.kieler.klighd.repository/pom.xml

```diff
@@ -32,10 +32,16 @@
             which has been built meanwhile, doesn't work properly yet,
             see https://bugs.eclipse.org/bugs/show_bug.cgi?id=453708#c34
             so we have to stick to this plug-in -->
+      <!-- Update: With https://github.com/eclipse-tycho/tycho/pull/145 (Tycho v2.4.0) the aforementioned issue is fixed.
+            However, just dropping the <associateSites>...</associateSites> argument and keeping the web content
+            generation doesn't work here, as this mojo _always_ clears the existing repository references,
+            or crashes if 'content.jar' doesn't exist. Hence, just binding it to an early build phase doesn't work.
+            Therefore I submitted PR https://github.com/jbosstools/jbosstools-maven-plugins/pull/88
+            to fix that, let's see how the committers assess it. -->
       <plugin>
         <groupId>org.jboss.tools.tycho-plugins</groupId>
         <artifactId>repository-utils</artifactId>
-        <version>1.0.0</version>
+        <version>2.5.0</version>
```

Comment:
What speaks against using the eclipse.tycho tycho-p2-repository-plugin here with the newer tycho release used in the build? From what I tested that should work as well without the need for the duplication of the associate sites here in the pom.xml

Reply:
I tried to skip that, but ... see https://github.com/kieler/KLighD/blob/3d54c4b3e684c1ee94546ce5269848e7bd0f3bd0/build/de.cau.cs.kieler.klighd.repository/pom.xml#L36-L40